database/migrations/2023_08_19_170326_remove_events_column_from_webhooks.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    public function up(): void {
+        Schema::table('webhooks', function (Blueprint $table) {
+            $table->removeColumn('events');
+        });
+    }
+
+    public function down(): void {
+        Schema::table('webhooks', function (Blueprint $table) {
+            $table->addColumn('uint', 'events');
+        });
+    }
+};
```

Comment:
Shouldn't this be accomplished with database/migrations/2023_08_19_174051_drop_webhook_events_column.php?

Reply:
yes, i don't know what i did here, was a bit late when i did that